js/deps.ts

```diff
@@ -1,24 +1,24 @@
-import React, { Fragment } from "https://esm.sh/react@17.0.2";
-import ReactDOM from "https://esm.sh/react-dom@17.0.2";
+import React, { Fragment } from "https://cdn.skypack.dev/react@17.0.2";
```

Comment:
It seems necessary to add `?dts` at the end of the url to get type definition from skypack

Reply:
Yea, packup index.html worked if I just added ?dts to the end but some of the modules were missing types due to issues on skypack.

After getting that working, I'm now trying to do SSR using react-dom server. When doing that I found out about the missing and broken types.

react-router-dom's types have an issue in skypack. I was able to kind of fix it manually but now having issues with react-dom server.

I'm thinking if I can't fix all the issues I'm having manually, I'll ditch tsx and try jsx without types. Then hopefully the issues with skypack and esm.sh will get fix d at some point.

Skypack also ignores version pinning and just always returns react 17.0.1. esm.sh respects it but ignores it for the types, when I use 17.0.2 it was giving me types for 17.0.35.